flink-ml-lib/src/main/java/org/apache/flink/ml/classification/knn/KnnParams.java

```diff
@@ -0,0 +1,32 @@
+package org.apache.flink.ml.classification.knn;
+
+import org.apache.flink.ml.common.param.HasFeatureColsDefaultAsNull;
+import org.apache.flink.ml.common.param.HasLabelCol;
+import org.apache.flink.ml.common.param.HasPredictionCol;
+import org.apache.flink.ml.common.param.HasVectorColDefaultAsNull;
+import org.apache.flink.ml.param.IntParam;
+import org.apache.flink.ml.param.Param;
+import org.apache.flink.ml.param.ParamValidators;
+import org.apache.flink.ml.param.WithParams;
+
+/** knn fit parameters. */
+public interface KnnParams<T>
+        extends WithParams<T>,
+                HasVectorColDefaultAsNull<T>,
+                HasLabelCol<T>,
+                HasFeatureColsDefaultAsNull<T>,
+                HasPredictionCol<T> {
+    /**
+     * @cn-name topK
+     * @cn topK
+     */
+    Param<Integer> K = new IntParam("k", "k", 10, ParamValidators.gt(0));
+
+    default Integer getK() {
```

Comment:
Just noticed that this `getK()` method is only used in tests, not in `Knn` algorithm. Is there any problem with the algorithm's implementation?

Reply:
K can be get in tow parttern: getK() or params.get(KnnParams.K)
algorithm's implementation is no problem. I just use the second pattern, maybe I need change the patten later.

Reply:
done